### ISSUE TITLE
Questions: change Update URL / integrate a DDNS-Service manually

### ISSUE THREAD

**Issue by RichardBeisser** [ddclient/ddclient#281]
Hello,

i am using ddclient v 3.9.1-5 on Raspberry PI 4 with Raspian sid.

1)
The tool already supports some ddns-Services so you dont have to set a update-URL manually.
Where can I see / change the update-URL for a integrated DDNS-Service?

Example: duckdns
There are different URL possible: https://www.duckdns.org/spec.jsp
https://www.duckdns.org/update?domains=&token={YOURVALUE}&txt={YOURVALUE}[&verbose=true][&clear=true]
https://www.duckdns.org/update?domains=&token={YOURVALUE}[&ip={YOURVALUE}][&ipv6={YOURVALUE}
https://www.duckdns.org/update?domains=&token={YOURVALUE}[&ip={YOURVALUE}][&ipv6={YOURVALUE}][&verbose=true][&clear=true]

Where can I change the used URL?

2)
Is it possible to integrate a ddns-service by myself which ddlcient do not support at the moment, f. ex. https://www.ddnss.de/ua/help.php

In other words, is it possible so use a own URL with placeholders in the URL for ID, Paswort, IP, etc?

Thanks

**Comment by dkerr64** [comment]
Your second request feels like a dup of #270
